source/dub/commandline.d

```diff
@@ -825,6 +825,83 @@ class RunCommand : BuildCommand {
 
 	override int execute(Dub dub, string[] free_args, string[] app_args)
 	{
+		import std.typecons : Nullable;
+
+		if (free_args.length >= 1)
+		{
+			auto package_name = free_args[0];
+			auto pack = dub.packageManager.getFirstPackage(package_name);
+
+			if (!pack)
```

Comment:
Please check some `--non-interactive` argument as well to prevent hanging scripts.